app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt

```diff
@@ -75,7 +75,7 @@ class MessageActivity : AppCompatActivity() {
                     presenter.sendMessage(this, convertToDeviceDataModel())
                 }
             } else {
-                Toast.makeText(this, getString(R.string.txt_turn_on_bluetooth), Toast.LENGTH_LONG).show()
+                Snackbar.make(content.rootView, getString(R.string.txt_turn_on_bluetooth), Snackbar.LENGTH_LONG).show()
```

Comment:
Do not replace this with a Snackbar. When this toast is shown, the keyboard will still be on the screen and hence the snackbar will not be visible. Let it be a Toast so that it is always visible.

Reply:
I'm not experiencing this problem, can you add a GIF showing this?